src/components/Error/index.js

```diff
@@ -7,7 +7,7 @@ import { localised } from '../../locales'
 
 class Error extends Component {
   componentDidMount() {
-    if (this.props.focusOnRender && this.container) {
+    if (this.props.focusOnMount && this.container) {
```

Comment:
it might be interesting to consider a higher order component to deal with this case of focusing on mount. the page title could use it as well. although this is minor